synapse/config/oidc_config.py

```diff
@@ -143,9 +143,9 @@ def generate_config_section(self, config_dir_path, server_name, **kwargs):
         #
         #           For the default provider, the following settings are available:
         #
-        #             sub: name of the claim containing a unique identifier for the
-        #                 user. Defaults to 'sub', which OpenID Connect compliant
-        #                 providers should provide.
+        #             subject_claim: name of the claim containing a unique identifier
```

Comment:
This was just documented wrong it looks like?

Reply:
yup.